tests/test_stump_topk.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import numpy.testing as npt
+import pandas as pd
+from stumpy import stump, config
+import pytest
+import naive
+
+
+test_data = [
+    (
+        np.array([9, 8100, -60, 7], dtype=np.float64),
+        np.array([584, -11, 23, 79, 1001, 0, -19], dtype=np.float64),
+    ),
+    (
+        np.random.uniform(-1000, 1000, [8]).astype(np.float64),
+        np.random.uniform(-1000, 1000, [64]).astype(np.float64),
+    ),
+]
+
+substitution_locations = [(slice(0, 0), 0, -1, slice(1, 3), [0, 3])]
+substitution_values = [np.nan, np.inf]
+
+
+def test_stump_int_input():
+    with pytest.raises(TypeError):
+        stump(np.arange(10), 5)
+
+
+@pytest.mark.parametrize("T_A, T_B", test_data)
+def test_stump_self_join_1NN(T_A, T_B):
```

Comment:
I don't think we need this in a new test file. This should just sit in `test_stump.py` and you'd be testing what happens when you call `stumpy.stump` with `k = 2` or `k > 1`.

Reply:
Yes. I think I get it now. I just need to add ONE NEW test function inside the EXISTING unit test file `test_stump.py` to check output for `k>1`.